app/lib/actions/middleware/keep_current_request_id.rb

```diff
@@ -42,13 +42,14 @@ def restore_current_request_id
         if (restored_id = action.input[:current_request_id]).present?
           old_id = ::Logging.mdc['request']
           ::Logging.mdc['request'] = restored_id
-          if old_id.present? && old_id != restored_id
-            action.action_logger.warn(_('Changing request id %{request_id} to saved id %{saved_id}') % { :saved_id => restored_id, :request_id => old_id })
```

Comment:
Do we not care about this?

Reply:
I was not able to get to a case where I would hit this, but it would probably not hut to be aware of this situation when it happens

Reply:
Updated